src/components/lib/sl/sl_thd_static_backend.c

```diff
@@ -9,9 +9,13 @@
 #include <consts.h>
 #include <ps.h>
 #include <cos_kernel_api.h>
+#include <cos_defkernel_api.h>
 
 static struct sl_thd_policy sl_threads[SL_MAX_NUM_THDS];
 
+static struct cos_aep_info __sl_aep_info[SL_MAX_NUM_THDS];
```

Comment:
either move `sl_threads` to `__sl_threads`, or move `__sl_aep_info` to `sl_aep_info`.  Consistency.

Should be `infos` to be consistent with `threads` (see plural).

Reply:
Missed that, I'll fix it. Thank you.

Reply:
Missed it, thank you. I'll make them consistent.